politeiawww/api/cms/v1/api.md

```diff
@@ -473,6 +473,98 @@ Reply:
   ]
 }
 ```
+
+### `Get comments`
```

Comment:
One thing that I think we did with pi that was a mistake was not be consistent across our naming convention  For example, whether to include `Get` in the GET request routes.  Most routes don't (ex. `ProposalDetails`, `UserDetails`, `VoteResults`), some routes do (`GetComments`).

My suggestion would be to not copy pi in this regard and keep the invoice naming convention consistent.  To me, names like `InvoiceComments` and `NewInvoiceComment` are more consistent with the existing naming convention.

Not a big deal either way though.  Your call.